pkg/inspector/image-inspector.go

```diff
@@ -35,6 +35,7 @@ const (
 	OWNER_PERM_RW            = 0600
 	HEALTHZ_URL_PATH         = "/healthz"
 	API_URL_PREFIX           = "/api"
+	RESULT_API_URL_PATH      = "/results"
```

Comment:
@simon3z maybe this is not the best URL path but this will provide results for both openscap and clam... wonder if we want to version this api (i this it is a good idea to do).